src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -385,6 +390,7 @@ public Integer invoke(File f, VirtualChannel channel) throws IOException, Interr
                         }
                     }
                 } catch (NumberFormatException x) {
+                    LOGGER.log(Level.WARNING, "corrupted content in {0} when reading with charset {1}: {2}", new Object[] {f, charset, x});
                     throw new IOException("corrupted content in " + f + ": " + x, x);
```

Comment:
Do not really want this to be in the system log. Just use
```suggestion
                    throw new IOException("corrupted content in " + f + " using " + charset + ": " + x, x);
```